endstate_correction/neq.py

```diff
@@ -11,18 +11,18 @@
 
 from endstate_correction.constant import distance_unit, temperature
 from endstate_correction.system import get_positions
-import openmm
 
 
 def perform_switching(
-    sim, lambdas: list, 
-    samples: list, 
-    nr_of_switches: int = 50, 
+    sim,
+    lambdas: list,
+    samples: list,
+    nr_of_switches: int = 50,
     save_trajs: bool = False,
-    save_endstates:bool = False,
+    save_endstates: bool = False,
 ) -> Tuple[list, list, list]:
     """performs NEQ switching using the lambda sheme passed from randomly dranw samples"""
```

Comment:
Do you mind add a doc string here to show what are these variables, please? particularly what is samples. Thank you!

Reply:
added doc string
the documentation is still lacking, I will reserve some time in the next days to update it